cms/static/js/views/course_info_helper.js

```diff
@@ -2,6 +2,7 @@ define(['codemirror', 'js/utils/handle_iframe_binding', 'utility'],
     function(CodeMirror, IframeBinding) {
         var editWithCodeMirror = function(model, contentName, baseAssetUrl, textArea) {
             var content = rewriteStaticLinks(model.get(contentName), baseAssetUrl, '/static/');
+            content = rewriteCdnLinksToStatic(model.get(contentName));
```

Comment:
Don't you think you need to replace `cdn` url before `rewriteStaticLinks`?

Reply:
@zaidbamber161 ^^

Reply:
I think there is no need of replacing cdn url before rewriteStaticLinks as both of them are replaced through different regex. Nature of working isn't affected.

Reply:
I think there is no need for replacing CDN URL before rewriteStaticLinks as both of them are replaced through the different regex. Nature of working isn't affected.

Reply:
I believe you have to pass `content` to `content = rewriteCdnLinksToStatic(content);` instead of `model.get(contentName)`, otherwise it will replace all the changes made here: https://github.com/edx/edx-platform/blob/560e9fe0eca9d11eed2e889d6ad3e6921a0f7ec7/cms/static/js/views/course_info_helper.js#L4